kinley-matt/styles/modules.css

```diff
@@ -0,0 +1,78 @@
+.site-header {
+  background: #FFEAAA;
+}
+
+.heading-group-h1 {
+  float: left;
+  padding: 0 0 .2em 1em;
+}
+
+.icon-menu:before {
+  content: "\e902";
```

Comment:
You do not need to have this `content` property because it is taken care of in the Icomoon CSS file. The only thing you need to make the Icomoon font work, in this case, is the class name on an HTML element.